src/site/stages/build/process-cms-exports/transformers/block_content-promo.js

```diff
@@ -2,8 +2,8 @@ const transform = entity => ({
   entity: {
     entityType: 'block_content',
     entityBundle: 'promo',
-    fieldImage: entity.fieldImage,
-    fieldPromoLink: entity.fieldPromoLink,
+    fieldImage: entity.fieldImage[0],
+    fieldPromoLink: entity.fieldPromoLink[0],
```

Comment:
Why did this change?

Reply:
The landing page uses the `block-content_promo` entity for `fieldPromo`, and I noticed that these two fields in `pages.json` were _actually_ objects instead of arrays.